### ISSUE TITLE
Figure out maintainers, teams/areas of responsibility, and timezone issues

### ISSUE THREAD

**Issue by alexkreidler** [openventilatorregistry/OpenVentilatorRegistry#29]
Hey all,

I just noticed that a lot of issues with requests to join were recently made and it seems the maintainer hasn't responded. This could likely be b/c of a timezone difference and sleep schedules.

I propose someone makes a Google Doc where collaborators list their names and the hours they can be open for working on the project, video calling, etc in UTC.

The doc could also organize people into teams to focus on frontend, backend, etc, but we could also do that with Github teams.

I also propose that the maintainer finds a group of others such that collectively one of them is up at all times (or at least covers as much of the day as possible) and adds them as a maintainer to manage new contributors and other questions, etc.

**Comment by alexkreidler** [comment]
It could also be a Google Sheet for all contributors or pending contributors and their skills (i'm realizing how big our Issues list is going to get just with request to join)

**Comment by jordan-rosenfeld** [comment]
@alexkreidler Would you be willing to add that to this projects wiki page?

https://github.com/openventilatorregistry/OpenVentilatorRegistry/wiki

**Comment by dacheson** [comment]
How do I sign up to help? Experienced web dev / designer / scrum manager. Familiar with building web apps with react/js and hosting in Azure. Happy to kick off some of the first web work items.

**Comment by jordan-rosenfeld** [comment]
@dacheson Invitation sent.

**Comment by supernoir** [comment]
Hey @jordan-rosenfeld would you consider giving a bunch of us admin access to help you out with maintaining the GH project? And please join the slack channel at openventilatorregis.slack.com. We wanted to add you but don't have your e-mail...

**Comment by jordan-rosenfeld** [comment]
@supernoir Yes I'm open to it.  For now would you mind joining the slack channel that I posted as the central channel for the organization on the homepage? I'm happy to use the channel you have above as well when we break out into teams, but for now let's continue the discussion there.

**Comment by supernoir** [comment]
Done!

**Comment by edco59** [comment]
In regards to the slack channel, does someone mind shooting me an add? My email is emily.carroll.59@gmail.com

**Comment by edco59** [comment]
If needed I wouldn’t mind compiling a google doc like the one being asked about in this main thread since I’m here to just do some grunt work like compiling stuff and getting research etc, so I could definitely handle this

**Comment by jamesperrin** [comment]
@supernoir  Please send me an invite to the slack channel.

**Comment by cafeTechne** [comment]
@supernoir I need an invite to the slack channel as well! 

cafetechne@gmail.com
